integrations/extensions/starter-kits/watson-discovery/README.md

```diff
@@ -0,0 +1,142 @@
+
+Note that unlike other starter kits such as the [Coveo search extension starter kit](https://github.com/watson-developer-cloud/assistant-toolkit/tree/master/integrations/extensions/starter-kits/coveo/README.md), this starter kit is somewhat customized to a particular application.  Specifically, it includes some illustrative examples of how to specialize your search by configuring your search differently for different user profiles or intents.  The example we use is the Bitcoin FAQ page mentioned in the "Pre-Requisite Steps" section above. Unless you want to build an assistant for that data, you will need to edit these actions to fit your use case or delete them.
+
+### Setup in a pre-existing Assistant
+
+If you want to add this starter kit to an _existing_ assistant, you cannot use the Actions JSON file since it will overwrite your existing configuration.  Furthermore, since this starter kit is somewhat specialized to the Bitcoin FAQ page example, it needs customizing to make work on another application.  Anyone wanting to add this capability to an existing bot may thus want to start by setting up the starter kit in a new Assistant (as described in the previous section), spend some time looking at how it works and what it does, and then apply what was learned to their own assistant.  Note that the OpenAPI specification (`watson-discovery-query-openapi.json`) is _not_ specialized for the Bitcoin example in any way, so you can use that to [build a custom extension](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension) without making any changes to it.  However, the Actions JSON is somewhat specialized.  In particular, for some other application you would want to discard or replace the following actions: "How many Bitcoins can exist?", "Bitcoin FAQ Search", and "Search within the mining topic".  You would also want to get rid of the first step of the general-purpose "Search" action which calls out to one of the "Bitcoin" actions under certain circumstances.
+
+### Setup the sample index.html file
+
+Included in this kit is an `index.html` file in the `client/javascript` subdirectory.  To use this file, do the following:
+
+- Setup the assistant (as described in "Setup in a new Assistant" above).
+- Open the web chat configuration screen (from the Environments or Integrations pages).
+- Select the "Embed" tab, which should look like the (redacted) image below:  
+![Web chat embed tab](./assets/web-chat-embed.png)  
+- From the included snippet, copy only the three lines starting with `integrationID`, `region`, and `serviceInstanceID`.  Don't copy the others, because the `index.html` file in this kit already has all that.
+- Edit the `index.html` file in this kit to replace those same three lines in the kit (which have dummy placeholder values) with the ones you copied in the previous step.  Now the script is pointing specifically to _your_ assistant.
+
+## Using this Starter Kit
```

Comment:
I think the title and the description below are a bit misleading.
One can use the starter kit without the index.html and the customization it provides.
The index.html should be under "Advanced Customization" section

Reply:
I agree with you that setting up index.html should be optional.  I tried restructuring things to move everything relating to index.html into a separate Advanced Customization section, but it created a lot of extra complexity because the document is already structured around setup vs. use, so I'd need to then split it into basic setup, basic use, advanced setup, and advanced use.  The filtering examples would probably go into the advanced section too.

I think all that could be done, but I think it would complicate the flow a lot, and the "Deciding whether to use this kit or the built-in search integration" section at the top makes it clear (I think) that the entire kit is for advanced users and that basic users should just use the built-in search integration.

So here is what I've done instead as a compromise:
1. I explicitly marked the parts that setup and use of `index.html` as "Advanced/Optional" to make it clear that users can skip these steps if they want.
2. I added an alternative "Using this Starter Kit" step and graphic showing how you can use it without index.html by going to the Preview tab in the product instead.

@amiddavid , do you think this is sufficient to address your concern?